src/utils/ipfs.ts

```diff
@@ -271,20 +271,14 @@ export const unpinIpfsFileByCid = (cid: string | undefined) =>
     : Promise.resolve(false)
 
 export const uploadProjectMetadata = (
-  metadata: Omit<ProjectMetadataV3, 'version'>,
+  metadata: Omit<ProjectMetadataV4, 'version'>,
 ) =>
-  pinJSONToIpfs(
-    {
-      ...metadata,
-      version: 3,
-    },
-    {
-      name: 'juicebox-project-metadata.json',
-      metadata: {
-        tag: IPFS_TAGS.METADATA,
-      },
+  pinJSONToIpfs(consolidateMetadata(metadata), {
```

Comment:
passing improvement, just ensures that anytime metadata is uploaded it's converted to the latest version